.devcontainer/Untitled-3.sh

```
@@ -0,0 +1,39 @@
+
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)